### ISSUE TITLE
possible to trigger this on file:// requests?

### ISSUE THREAD

**Issue by jzacsh** [jscher2000/Content-Type-Fixer-extension#11]
this _might_ be a generic firefox-addons question, but maybe not... is it possible for me to override the behavior of **firefox's own** behavior of `.md` files? I found your reply in [this mozilla forum thread](https://support.mozilla.org/en-US/questions/1280325), but that's discussing http requests. In chrome I'd often visit `~/TODOs.md` and then a separate markdown extension would render that text nicely for me. When I visit `file:///home/myuser/TODOs.md` in firefox however, it tries to download it (as if served by a server with an HTTP header `content-disposition: attachment`).

**tl;dr** When I checkout the _"Log of Recent Request"_ page for this extension, I don't see any of the requests triggered by my `file://` browsing.. Is there a button I have to click somewhere to give the addon access to this traffic?
